stable/external-dns/values.yaml

```diff
@@ -74,7 +74,9 @@ aws:
   ## AWS Zone tags
   ##
   zoneTags: []
-
+  ## Enable AWS Prefer CNAME. Available values are: true, false
+  ##
+  preferCNAME: ""
```

Comment:
If the previous comment doesn't apply and it is just a flag (without value), is fine maintaining the _values.yaml_ as is, but here you need to use `true`/`false` as the value of this parameter, and then, check in the _values.yaml_ this condition (you're doing it properly right now) to add the flag without assignment.

Reply:
The above comment applies here 
```
This parameter doesn't need value as it is just a flag, but the user should have the flexibility to either choose it or not, so I added it a key/value with the boolean operators in the values.yaml file.
```